scripts/docker/buildpack-deps/Dockerfile.ubuntu2004

```diff
@@ -47,15 +47,11 @@ FROM base AS libraries
 
 # EVMONE
 RUN set -ex; \
-	cd /usr/src; \
-	git clone --branch="v0.9.1" --recurse-submodules https://github.com/ethereum/evmone.git; \
-	cd evmone; \
-	mkdir build; \
-	cd build; \
-	cmake -G Ninja -DBUILD_SHARED_LIBS=ON -DCMAKE_INSTALL_PREFIX="/usr" ..; \
-	ninja; \
-	ninja install/strip; \
-	rm -rf /usr/src/evmone
+	wget -O /usr/src/evmone.tar.gz https://github.com/ethereum/evmone/releases/download/v0.10.0/evmone-0.10.0-linux-x86_64.tar.gz; \
```

Comment:
Why for this image you decided to not build from the source?

Reply:
Because evmone switched to requiring C++20, which ubuntu 20.04 does not have a suitably capable compiler for.